tests/target/imports_granularity_item.rs

```diff
@@ -0,0 +1,13 @@
+// rustfmt-imports_granularity: Item
+
+use a::b;
+use a::c;
+use a::d;
+use a::f::g;
+use a::h::i;
+use a::h::j;
+use a::l;
```

Comment:
As discussed in #4641, this should be `a::l::{self}`, no?

Reply:
Ah, you're right. I added the above transformation because this was coming out as `use a::l::self;`, which is invalid.